pkg/clients/clients.go

```diff
@@ -201,8 +203,17 @@ func NewClients(configPath, clusterName, namespace string) *Clients {
 	c.PipelineRunClient = cs.TektonV1beta1().PipelineRuns(namespace)
 	c.PipelineResourceClient = rcs.TektonV1alpha1().PipelineResources(namespace)
 	c.ConditionClient = cs.TektonV1alpha1().Conditions(namespace)
-	c = initTestingFramework(c)
-	return c
+	c, err = initTestingFramework(c)
+
+	// When we don't subscribe to Operator initTestFramework will throw error waiting for the condition (config to be present on default namespace)
+	if err != nil {
+		if strings.Contains(strings.ToLower(os.Getenv("INSTALL_OPERATOR")), "y") {
+			log.Printf("WARNING: Pipelines not available: %+v, waiting for operator to be installed......", err)
```

Comment:
It says "waiting for operator..." but it doesn't do anything, where can I find the waiting logic?